cmake/Modules/FindQVTK.cmake

```diff
@@ -10,6 +10,7 @@
 if(";${VTK_MODULES_ENABLED};" MATCHES ";vtkGUISupportQt;" AND ";${VTK_MODULES_ENABLED};" MATCHES ";vtkRenderingQt;")
   set(VTK_USE_QVTK ON)
   set(QVTK_LIBRARY vtkRenderingQt vtkGUISupportQt)
+  list(APPEND PCL_VTK_COMPONENTS vtkRenderingQt vtkGUISupportQt)
 else()
   unset(QVTK_FOUND)
```

Comment:
`QVTK_FOUND` will by only unset, but never set.

Reply:
Hm, but it's not used anywhere in the project, so I suppose I'll just drop this.

Reply:
Or, I'd rather drop `VTK_USE_QVTK` and switch to using `VTK_FOUND` in other places.